### PR TITLE
use uncover api to scan thro the proofs

### PR DESCRIPTION
**Pull request by KarishmaBothara**

### CODE REVIEW

File: scripts/scanEventsFromUncover.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    // Get the bridge instance that was deployed
+    const Bridge = await ethers.getContractFactory('CENNZnetBridge');
+    logger.info('Connecting to CENNZnet bridge contract...');
+    const bridge = await Bridge.attach(bridgeContractAddress);
+    await bridge.deployed();
+    logger.info(`CENNZnet bridge deployed to: ${bridge.address}`);
+    logger.info(`Executor: ${txExecutor.address}`);
+
+
+    const response = await axios.get(
```
Comment (KarishmaBothara) [inline scripts/scanEventsFromUncover.js:105]: everything remains same, except for the events here are fetched from uncover api and are then submitted

**Review by holygits — APPROVED** [review]
does this check to see if contract has the event already?

**Comment by KarishmaBothara** [comment]
> does this check to see if contract has the event already?

yes.. it does check at https://github.com/cennznet/bridge-contracts/blob/e289740e55c622162f697698913c593ce882c8f5/scripts/scanEventsFromUncover.js#L37
check if event exist in bridge contract before trying to submit the proof

File: scripts/scanEventsFromUncover.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        `${process.env.UNCOVER_BASEPATH}/cennznet-explorer-api/api/scan/events?moduleId=ethBridge&eventId=AuthoritySetChange&row=100&page=0`
```
Comment (holygits) [inline scripts/scanEventsFromUncover.js:106]: would call it `UNCOVER_URI` for consistency

**Pull request merged**